### PR TITLE
hub land will squash+merge the pr via github api

### PR DESCRIPTION
**Pull request by robbiegill**
- instead of closing the pr

### CODE REVIEW

File: lib/hub/commands.rb [hunk 1]
```diff
@@ -343,21 +320,43 @@ def land(args)
         exit 1
       end
 
-      title, body, url = pr['title'], pr['body'] || '', pr['html_url']
+      commit_title, commit_message, pr_number = pr['title'], pr['body'] || '', pr['number']
```
Comment (nickgervasi) [inline lib/hub/commands.rb]: Nit - split into three lines?

File: lib/hub/commands.rb [hunk 2]
```diff
@@ -343,21 +320,43 @@ def land(args)
...
+        status = response.status
+        $stderr.puts '-' * 80
+        if status == 409
+          $stderr.puts 'Error landing PR. Local commit sha does not match the Github remote.'
+          $stderr.puts ''
+          $stderr.puts 'You have probably made changes locally:'
+          $stderr.puts ' - push'
+          $stderr.puts ' - wait for ci'
+          $stderr.puts ' - land again'
+        elsif status == 405
```
Comment (nickgervasi) [inline lib/hub/commands.rb]: nit, but since both are `status ==` it's a good use case for `case/when/else`.

**Comment by zp-alfred** [comment]
Accepted by nickgervasi!

**Comment by zp-alfred** [comment]
Accepted by nickgervasi!

**Pull request merged**